UM/Platform.py

```diff
@@ -1,41 +1,33 @@
 # Copyright (c) 2015 Ultimaker B.V.
 # Cura is released under the terms of the LGPLv3 or higher.
 
+import collections
 import sys
 
-
 ##  Convenience class to simplify OS checking and similar platform-specific handling.
 class Platform:
-    class PlatformType:
-        Windows = 1
-        Linux = 2
-        OSX = 3
-        Other = 4
+
+    ##  Get the platform type.
+    @staticmethod
+    def getType() -> int:
+        platforms = collections.defaultdict(lambda: '4')  # Return 4 if the OS is not Windows, Linux or OSX
+        platforms = {'win32': 1,    # Windows
+                     'linux': 2,    # Linux
+                     'darwin': 3,   # OSX
+                    }
+        return platforms[sys.platform]
 
     ##  Check to see if we are currently running on OSX.
-    @classmethod
-    def isOSX(cls) -> bool:
-        return cls.__platform_type == cls.PlatformType.OSX
+    @staticmethod
+    def isOSX() -> bool:
+        return sys.platform == 'darwin'
 
     ##  Check to see if we are currently running on Windows.
-    @classmethod
-    def isWindows(cls) -> bool:
-        return cls.__platform_type == cls.PlatformType.Windows
+    @staticmethod
+    def isWindows() -> bool:
+        return sys.platform == 'win32'
```

Comment:
```suggestion
        return sys.platform == "win32"
```
We always use double quotes for strings.